example/docker-compose.extra.yml

```diff
@@ -0,0 +1,102 @@
+# Wikibase for testing
+version: '3'
+
+services:
+  elasticsearch:
+    image: "${ELASTICSEARCH_IMAGE_NAME}"
+    restart: always
+    networks:
+      default:
+        aliases:
+         - elasticsearch.svc
+    environment:
+      discovery.type: single-node
+      ES_JAVA_OPTS: "-Xms512m -Xmx512m"
+  wdqs-frontend:
+    image: "${QUERYSERVICE_UI_IMAGE_NAME}"
+    restart: always
+    ports:
+    # CONFIG - Change the 8282 here to expose the Query Service UI on a different port
```

Comment:
Should say `8834`.

Reply:
removed this whole thing as it moved to the env